solutions/web3-sessions/README.md

```diff
@@ -0,0 +1,53 @@
+# web3-sessions example
+
+This example shows how create sessions in Next.js with [Metamask](https://metamask.io/) and [NextAuth.js](https://next-auth.js.org/).
+
+## Demo
+
+to do update
+
+https://web3-sessions.vercel.app
+
+This Demo will show how to configure NextAuth.js and create a custom hook to protect routes.
+
+## How to Use
+
+You will need to add the proper variables in a `.env` file like show in the `.env.example` file.
+
+```
+// generate random secrets on https://passwordsgenerator.net/ or anywhere else similar
+NEXT_AUTH_SECRET=123
+JWT_SECRET=123
```

Comment:
Let's not use `123` as the example value here but a long and secure password instead.

Reply:
Good point!